Update Add Token guides and add bridging guide (#92)
LiskHQ/lisk-docs: docs/building-on-lisk/add-token-to-lisk/index.md

    'Lisk Mainnet',
    'Optimism Superchain token list',
  ]
---

> import DocCardList from '@theme/DocCardList';

Reviewer: As we already link to the relevant guides, adding cards also seems redundant. Please remove them.
```suggestion
```

To add your token to the Lisk Token list, perform the following steps. -->

> ## [Deploying a Standard ERC-20 Token](./standard-token.md)

Reviewer: ```suggestion
## [Deploying a Standard ERC-20 token](./standard-token.md)
```

> To deploy a Standard ERC-20 Token to Lisk Sepolia, follow the steps mentioned in the [Add a Standard ERC-20 Token to Lisk](./standard-token.md) guide.

Reviewer: ```suggestion
To deploy a Standard ERC-20 Token to Lisk Sepolia, follow the steps mentioned in the [Deploying Your Standard ERC-20 Token to Lisk](./standard-token.md) guide.
```

The guide explains how to use the [OptimismMintableERC20Factory](https://github.com/ethereum-optimism/optimism/blob/186e46a47647a51a658e699e9ff047d39444c2de/packages/contracts-bedrock/contracts/universal/OptimismMintableERC20Factory.sol) to deploy a standardized ERC-20 token on Lisk Sepolia.

> Tokens created by this factory contract implement the `IOptimismMintableERC20` interface and include basic logic for deposits, transfers, and withdrawals.

Reviewer: ```suggestion
Tokens created by this factory contract implement the `IOptimismMintableERC20` interface, including basic logic for deposits, transfers, and withdrawals.
```

## [Deploying a Custom ERC-20 token](./custom-token.mdx)

A custom token allows you to do things like trigger extra logic whenever a token is deposited.

> To deploy a custom ERC-20 token to Lisk Sepolia, follow the steps mentioned in the [Add a Custom ERC-20 Token to Lisk](./custom-token.mdx) guide.

Reviewer: ```suggestion
To deploy a custom ERC-20 token to Lisk Sepolia, follow the steps mentioned in the [Deploying your Custom ERC-20 token to Lisk](./custom-token.mdx) guide.
```

The guide explains how to implement all the functions defined by the [IOptimismMintableERC20](https://github.com/ethereum-optimism/optimism/blob/v1.1.4/packages/contracts-bedrock/src/universal/IOptimismMintableERC20.sol) interface such as the logic for the `mint`, `burn`, `remoteToken`, and `bridge` functions of the `IOptimismMintableERC20` interface.

> <DocCardList />

Reviewer: ```suggestion
```
